assets/src/application/ui/layout/espressoTable/TableFooter.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import warning from 'warning';
+
+import { isEmpty } from '@appServices/utilities/array';
+import { isFunc } from '@appServices/utilities/function';
+
+import TableRow from './TableRow';
+import TableHeaderCell from './TableHeaderCell';
+
+import { RowType, TableFooterProps } from './types';
+
+const TableFooter: React.FC<TableFooterProps> = ({ tableId, footerRows, rowCount, ...props }) => {
+	const className = classNames(props?.className?.footerClassName, 'ee-rspnsv-table-footer');
+
+	return !isEmpty(footerRows) ? (
+		<tfoot className={className}>
+			{footerRows.map((footerRow, index) => {
+				const row = index + rowCount;
+
+				return (
+					<TableRow
+						rowData={footerRow}
+						key={`row-${row}`}
+						rowNumber={row}
+						rowType={RowType.footer}
+						htmlId={footerRow.id || tableId}
+						rowClassName={footerRow.footerRowClassName}
+						className={props.className}
+					>
+						{footerRow.cells.map((column, col) => {
+							warning(
+								column.hasOwnProperty('value'),
+								`Missing "value" property for footer column ${col}.`
+							);
+
+							return isFunc(column.render) ? (
+								column.render({ row, col, column })
+							) : (
+								<TableHeaderCell
+									key={`row-${row}-col-${col}`}
+									rowNumber={row}
+									colNumber={col}
+									rowType={RowType.footer}
+									id={column.id || tableId}
```

Comment:
if `id` is not provided, then table id and and cell id will be same.

Reply:
updated already :+1: